docker-compose.yml

```diff
@@ -63,7 +63,6 @@ services:
     depends_on:
       - db
       - cache
-      - worker
```

Comment:
I had this here so the worker would start if you say `docker-compose up backend`. I'm fine with removing it, but then you either need to start the whole thing or select a longer list.

Reply:
I amended the docs to emphasize this behaviour.